graphics/opengl/shader.cpp

```diff
@@ -33,6 +33,10 @@ namespace OpenGL {
 static const GLchar *readFile(const Common::String &filename) {
 	Common::File file;
 
+	#if defined(EMSCRIPTEN)
```

Comment:
you can not do that. this is only for development purpose.

Reply:
I actually forgot about removing this. I just tried to adjust this and I still struggle though about how to handle these. For some reason non of the shaders are found although they are all placed in the subdirectory `shaders/`. I quickly checked the macOS nightly here and that seems to be the same way there (the individual shaders aren't in subdirectories but flat as they are placed in `data/shaders` by `make dist-generic`). Weirdly enough all other assets are loaded just fine, so I'm really at a loss here why this doesn't work.

Reply:
_deleted my previous comment as it's not relevant anymore_

This is now fixed. Thank you for pointing out!